src/libostree/ostree-deployment.c

```diff
@@ -87,6 +87,32 @@ ostree_deployment_get_bootserial (OstreeDeployment *self)
   return self->bootserial;
 }
 
+char *
+_ostree_deployment_get_version (OstreeDeployment  *self,
+                                OstreeRepo        *repo,
+                                GError           **error)
```

Comment:
Error-handling for this function is funky. `NULL` could mean an error occurred or there is no version. Since this is opportunistic anyway and all callers need to gracefully deal with missing version strings, maybe simpler to just drop the `error` arg entirely instead of having them now check if `error` was set.